### DISCUSSION TITLE
🚀 Marp for VS Code has reached over 100K installs! 🎉

### DISCUSSION THREAD

**Discussion by yhatt** [marp-team/marp#264] [Announcements]
[![](https://user-images.githubusercontent.com/3993388/152623603-1e43a57c-90b8-4764-b345-3bccf47e3ac4.png)](https://twitter.com/y_hatt/status/1489599567251181571)

https://twitter.com/y_hatt/status/1489599567251181571
https://marketplace.visualstudio.com/items?itemName=marp-team.marp-vscode

> A VS Code extension is the easiest interface to start writing Markdown presentation with our Marp ecosystem. This is awesome milestone since the beginning of my Marp project! ◢◢
>
> &mdash; Yuki Hattori (@y_hatt) [February 4, 2022](https://twitter.com/y_hatt/status/1489601124738822146)

**Comment by chenxizhang** [comment]
おめでとう  
恭喜
Congratulations!!!
